CONTRIBUTING.md

```diff
@@ -15,56 +15,54 @@ that we follow.
-template or if your issue is something else, please use the default template.
-Please include as much detail as possible and try to remove the extra stuff
-that doesn't really relate to the issue itself. The easier it is for us to
-reproduce it, the faster it'll be fixed!
+Before reporting an issue, check our backlog of [open issues](https://github.com/containers/podman/issues) to see if someone else has already reported it.
+If so, feel free to add your scenario, or additional information, to the discussion.
+Or simply "subscribe" to it to be notified when it is updated.
+
+Note: Older closed issues/PRs are automatically locked.
+If you have a similar problem please open a new issue instead of commenting.
+
+If you find a new issue with the project we'd love to hear about it!
+The most important aspect of a bug report is that it includes enough information for us to reproduce it.
+To make this easier, there are three types of issue templates you can use.
+If you have a bug to report, please use *Bug Report* template.
+If you have an idea to propose, please use the *Feature Request* template.
+If your issue is something else, please use the default empty template.
+Please include as much detail as possible and try to remove anything that doesn't really relate to the issue itself.
+The easier it is for us to reproduce it, the faster it'll be fixed!
 
 Please don't include any private/sensitive information in your issue!
+Security issues should NOT be reported via Github and should instead be reported via the process described in our [README](https://github.com/containers/podman/blob/main/README.md#communications).
 
 ## Working On Issues
 
-Once you have decided to contribute to Podman by working on an issue, check our
-backlog of [open issues](https://github.com/containers/podman/issues) looking
-for any that do not have an "In Progress" label attached to it.  Often issues
-will be assigned to someone, to be worked on at a later time.  If you have the
-time to work on the issue now add yourself as an assignee, and set the
-"In Progress" label if you’re a member of the “Containers” GitHub organization.
-If you can not set the label, just  add a quick comment in the issue asking that
-the “In Progress” label be set and a member will do so for you.
+Once you have decided to contribute to Podman by working on an issue, check our backlog of [open issues](https://github.com/containers/podman/issues) looking for any that are unassigned.
+If you want to work on a specific issue that is already assigned, but does not appear to be actively being worked on, please ping the assignee in the issue and ask if you can take over.
+If they do not respond after several days, you can ping a maintainer to have the issue reassigned.
+When working on an issue, please assign it to yourself.
+If you lack permissions to do so, you can ping the `@containers/podman-maintainers` group to have a maintainer set you as assignee.
```

Comment:
The openshift bit should be able to assign via a `/assign` comment which does not need permission from the user AFAIK, maybe test that and document that so we do not have to deal with it.